### PR TITLE
Modified the calibration scripts which are cherry-picked from branch v3.1_dev and the environment variable

### PR DESCRIPTION
**Pull request by lishuanghua**
Modified the calibration scripts which are cherry-picked from branch v3.1_dev and the environment variable

### CODE REVIEW

File: modules/tools/calibration/process.py [hunk 1]
```diff
@@ -48,12 +48,12 @@ def get_start_index(data):
...
 
-    start_ind = np.where(data['brake_percentage'] == 40)[0][0]
+    start_ind = np.where(data['vehicle_speed'] == 0.0)[0][0]
 
     ind = start_ind
     while ind < len(data):
-        if data['brake_percentage'][
-                ind] == 40:  #or data['vehicle_speed'][ind] == 0.0:
+        if data['vehicle_speed'][
+                ind] == 0.0:  #or data['vehicle_speed'][ind] == 0.0:
```
Comment (xiaoxq) [inline modules/tools/calibration/process.py]: 1. Fix coding style, put comments in another line.
2. Comparing float numbers is not safe.

File: modules/tools/calibration/process_data.sh [hunk 2]
```diff
@@ -18,8 +18,9 @@
 
 #! /bin/bash
 
-rm result.csv
-for f in `ls ${1}/*_recorded.csv`
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" && pwd  )"
```
Comment (xiaoxq) [inline modules/tools/calibration/process_data.sh]: Remove redundant spaces.

File: modules/tools/calibration/process.py (hunk 1 above)
Comment (lishuanghua) [reply modules/tools/calibration/process.py]: I have already modified them, please review them again. Think you!

File: modules/tools/calibration/process_data.sh (hunk 2 above)
Comment (lishuanghua) [reply modules/tools/calibration/process_data.sh]: I have already modified them, please review them again. Think you!

**Pull request merged**